sql/sql_yacc.yy

```diff
@@ -1263,21 +1257,21 @@ void warn_about_deprecated_binary(THD *thd)
 /*
    Tokens from Percona Server 5.7 and older
 */
-%token<lexer.keyword> CHANGED_PAGE_BITMAPS_SYM
-%token<lexer.keyword> CLIENT_STATS_SYM
-%token CLUSTERING_SYM
-%token<lexer.keyword> COMPRESSION_DICTIONARY_SYM
-%token<lexer.keyword> INDEX_STATS_SYM
-%token<lexer.keyword> TABLE_STATS_SYM
-%token<lexer.keyword> THREAD_STATS_SYM
-%token<lexer.keyword> USER_STATS_SYM
-%token<lexer.keyword> ENCRYPTION_KEY_ID_SYM
+%token<lexer.keyword> CHANGED_PAGE_BITMAPS_SYM 1300
+%token<lexer.keyword> CLIENT_STATS_SYM 1301
+%token CLUSTERING_SYM 1302
+%token<lexer.keyword> COMPRESSION_DICTIONARY_SYM 1303
+%token<lexer.keyword> INDEX_STATS_SYM 1304
+%token<lexer.keyword> TABLE_STATS_SYM 1305
+%token<lexer.keyword> THREAD_STATS_SYM 1306
+%token<lexer.keyword> USER_STATS_SYM 1307
+%token<lexer.keyword> ENCRYPTION_KEY_ID_SYM 1308
 
 /*
    Tokens from Percona Server 8.0
 */
-%token<lexer.keyword> EFFECTIVE_SYM
-%token  SEQUENCE_TABLE_SYM
+%token<lexer.keyword> EFFECTIVE_SYM 1350
```

Comment:
This will result with different digest as before it was 1309

Reply:
@kamil-holubicki Yes, I agree.
However, this is a sacrifice I am willing to make deliberately to introduce a free range for future 5.7 extensions